packages/open-payments/src/client/index.ts

```diff
@@ -296,3 +339,42 @@ export const createAuthenticatedClient = async (
     })
   }
 }
+
+export const createMonetizationClient = async (
```

Comment:
do we want to call this `monetizationClient` directly? I'm going a bit back and forth on this since it's not specifically related to WM, but would only probably be used in WM contexts...
Something like `createCustomAuthenticatedClient`? Although thats a little wordy...